package.json

```diff
@@ -17,7 +17,9 @@
     "css:stats": "analyze-css --file build/bitstyles.css --pretty > test/css-stats.json",
     "precss:graph": "yarn build",
     "css:graph": "specificity-graph build/bitstyles.css --output test/specificity_graph/",
-    "build-storybook": "build-storybook"
+    "build-storybook": "yarn build && build-storybook",
+    "predeploy-storybook": "yarn build-storybook",
+    "deploy-storybook": "gh-pages -d storybook-static"
```

Comment:
Not sure about the exact workflow you had in mind for this, but one could even use a Github Action (e.g. [this](https://github.com/marketplace/actions/deploy-to-github-pages)) to automatically update these docs 🤔

Reply:
That action looks perfect, thanks! If we use it, there’d be no need for the `gh-pages` package, nor any of the deploy scripts, right? 🤔

Reply:
I didn't dive into the details of it. But I'd expect it to work like that, yes.